src/main/java/seedu/address/logic/commands/DeleteContactFromTeamCommand.java

```diff
@@ -0,0 +1,80 @@
+package seedu.address.logic.commands;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.contact.Contact;
+import seedu.address.model.team.Team;
+
+import static java.util.Objects.requireNonNull;
+
+public class DeleteContactFromTeamCommand extends Command {
+
+        public static final String COMMAND_WORD = "delete-from-team";
+
+        public static final String MESSAGE_USAGE = COMMAND_WORD + ": Deletes a contact from the team. "
+                + "Parameters: "
+                + "Team index number, contact index number \n"
+                + "Example: delete-from-team 2 3";
+
+        public static final String MESSAGE_SUCCESS = "Contact removed from team: %1$s";
+        public static final String MESSAGE_INEXISTENT_CONTACT_IN_TEAM = "A contact with this index does not exist in " +
+                "the team";
+
+        private final int teamIndex;
+        private final int contactIndex;
+
+        /**
+         * Creates a DeleteContactFromTeamCommand to delete the specified {@code Contact}
+         */
+        public DeleteContactFromTeamCommand(int teamIndex, int contactIndex) {
```

Comment:
You should use `Index` instead of `int`, this would also save you from having to manually subtract 1 to get the correct index.